aii-core/src/main/perl/aii-shellfe

```diff
@@ -491,6 +493,10 @@ sub app_options
          HELP    => 'Do not update OS installer (e.g. kickstart) configuration',
          DEFAULT => undef },
 
+       { NAME    => "$PROTECTED_OPTION=s",
+         HELP    => 'confirm id for reinstalling protected nodes',
```

Comment:
Is there a more detailed explanation of how this works? i.e. how do I figure out what the correct id is? Is is the FQDN of the host or something else?